applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -531,7 +531,7 @@ Every Bluetooth LE bridged device uses a separate Bluetooth LE connection, so yo
 Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
 
 Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
-The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``9``.
```

Comment:
This is not true. @ArekBalysNordic you have changed only the number of connections for nRF5340, but the default value in prj.conf for all platforms is still 10.

Reply:
Let's wait for the stack changes explanation, maybe it is not needed at all.